.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
Tego pliku też nie chcemy. Ma on być u Ciebie na dystku. ale cały katalog ma być dodany do .gitignore, bo nie chcemy mieć twoich ustawień w repo - jeśli nie było takiej decyzji, że synchronizujemy ustawienia VSC.

1. Dodaj katalog `.vscode` do `.gitignore`
2. Usuń katalog `.vscode` ale zostaw go na dysku: `git rm --cached -r .vscode` (więcej o tej komendzie tu: https://devconnected.com/how-to-clear-git-cache )
3. Dodaj do commita zmiany i wypchnij